.teamcity/Installers/Templates/PackageAndPublishInstallerLibrary.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed to the Ed-Fi Alliance under one or more agreements.
+ * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ * See the LICENSE and NOTICES files in the project root for more information.
+ */
+package installers.templates
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
```

Comment:
see above about the consistency of package imports.

Reply:
`import jetbrains.buildServer.configs.kotlin.v2019_2.*` is present in all files created when exporting from TeamCity. In a real Java/Kotlin product where you're producing a jar, I would agree about trying to optimize away the `*` on line 9. Trying to remove this import and figure out all of the correct ones would be rather time-consuming and would not help provide any benefit that I know of when loading the configurations into TeamCity.